content/en/tracing/setup_overview/custom_instrumentation/dotnet.md

```diff
@@ -120,6 +171,10 @@ using (var parentScope =
 }
 ```
 
+## Trace client and Agent configuration
+
+There are additional configurations possible for both the tracing client and Datadog Agent for context propagation and for excluding specific Resources from sending traces to Datadog in the event these traces are not wanted to count in metrics calculated, such as Health Checks.
```

Comment:
This line was based on the language in the Java documentation, though I think it can be improved. I'm happy to hear any suggestions/recommendations on this

Reply:
Or you can take this line out? It is largely covered by the section just below on Resource Filtering.

Reply:
Removed 👍🏼